File: EASY/REVERSE-DIGITS/rd.cpp
#include <string>
#include <iostream>
using namespace std;

class Solution {
    public:
    int reverseDigit(int n){
        string s = to_string(n);
        reverse(s.begin(), s.end());
        return stoi(s);
    }
};

int main(){
    Solution s;
    cout << s.reverseDigit(12345) << endl;  // Output: 54321
    return 0;
}

File: EASY/REVERSE-DIGITS/rd.java
class Solution {
    public int reverseDigit(int n){
        return Integer.parseInt(new StringBuilder(Integer.toString(n)).reverse().toString());
    }
}

class Main {
    public static void main(String[] args) {
        Solution s1 = new Solution();
        System.out.println(s1.reverseDigit(1234));
    }
}